### ISSUE TITLE
How to enable closing of calendar when the same date is chosen in kendo-datepicker Angular?

### ISSUE THREAD

**Issue by NiceStepUp** [telerik/kendo-angular#1275]
## I'm submitting an enhancement.

* Suggestion for improvement

## Current behavior
I have a <kendo-datepicker/>. It works perfectly, however it does not give a chance to select the previous date twice.

For example, if I choose the following date 1.1.2018, then <kendo-datepicker/> closes with selected date. It is okay.

However, if I want to select the same date 1.1.2018, then <kendo-datepicker/> does not close with selected date.

https://www.telerik.com/kendo-angular-ui/components/dateinputs/datepicker/#toc-calendar-templates

## Expected behavior
What I want is just always have a chance to choose the previously selected date.

## Minimal reproduction of the problem with instructions

You can reproduce this behaviour at Basic Usage of Overview of kendo-datepicker:

https://www.telerik.com/kendo-angular-ui/components/dateinputs/datepicker/#toc-calendar-templates

## What is the motivation or use case for changing the behavior?

I just would like to select the same date which is previously chosen.

## Environment

Package versions:

"@ progress/kendo-angular-dateinputs": "1.4.1",```

```

Browser:
<!-- Leave only the browsers that you tested the issue with. -->
- Chrome (desktop) version Version 63.0.3239.132 
- IE version 11

**Comment by NiceStepUp** [comment]
As a workaround I decided to handle click event at the day of calendar and use toggle method on kendo-datepicker to collapse calendar.

So we have kendo-datepicker:

`<kendo-datepicker #kendoDataPicker>
    <ng-template kendoCalendarCellTemplate let-date>
        <div (click)="handleClickCell(kendoDataPicker, date)">
            {{date.getDate()}}
        </div>
    </ng-template>
</kendo-datepicker>
`

and JavaScript code:

`handleClickCell(kendoDataPicker : any, selectedDate: any){
    if( kendoDataPicker.hasOwnProperty('_value')
        && typeof( kendoDataPicker._value)!=='undefined'
        && typeof( selectedDate)!=='undefined'
        && selectedDate.toString() === kendoDataPicker._value.toString()
      ){
            kendoDataPicker.toggle();
    }
}`

It is so cool that `toggle()` method of `kendo-datepicker` method can work out whether calendar should be collapsed or not.

**Comment by ggkrustev** [comment]
Hi @NiceStepUp,
thanks for the suggestion and for the shared workaround. It definitely will be beneficial to close the popup on every click not only on change.
We will plan the implementation for some of our future releases. We will update the thread once a dev version is available.

**Comment by ggkrustev** [comment]
A fix is released in `v3.4.0-dev.201805210833`. The easiest to try the change is to install the `DEV` release:
`npm i @progress/kendo-angular-dateinputs@dev`
